app/virtual-device/pods/common/index.js

```diff
@@ -1,16 +1,19 @@
 'use strict';
 
 var angular = require('angular');
+require('angular-uuid');
 
 var clamp = require('./values/clamp');
 var csvParse = require('./values/csv-parse');
 var randomInteger = require('./values/random-integer');
 var wiggle = require('./factories/wiggle');
+var deviceLogService = require('./factories/device-log-service-factory');
 var applicationConfig = require('../../../manage/pods/common/utils/application-config-constant');
 var loginProvider = require('../../../manage/pods/common/login-provider');
 var BlueprintClientProvider = require('../../../manage/pods/common/blueprint-client-provider');
+var states = require('./values/states');
 
-var commonModule = angular.module('common', []);
+var commonModule = angular.module('common', ['angular-uuid']);
```

Comment:
this should be fine, most of the 3rd party modules return their name: `[require('angular-uuid')]`